README.md

```diff
@@ -9,6 +9,9 @@ Group members:
 - Raymond Sarinas
 - Christian Lo
 
+# Deployment Link
+Our project is not deployed on Heroku yet as we plan to deploy it for project part 2. It is currently only runnable locally.
```

Comment:
plus "We will update this section with the deployment link once deployed on Heroku."